be/src/vec/aggregate_functions/aggregate_function_sort.h

```diff
@@ -87,7 +87,7 @@ struct AggregateFunctionSortData {
 
         PBlock pblock;
         pblock.ParseFromString(data);
-        block = Block(pblock);
```

Comment:
It is not equal.
Maybe should create new block and deserialize and call operator =